plugin-gradle/src/main/java/com/diffplug/gradle/spotless/SpotlessExtension.java

```diff
@@ -193,6 +193,12 @@ public void json(Action<JsonExtension> closure) {
 		format(JsonExtension.NAME, JsonExtension.class, closure);
 	}
 
+	/** Configures the special YAML-specific extension. */
+	public void yaml(Action<YamlExtension> closure) {
+		requireNonNull(closure);
+		format(JsonExtension.NAME, YamlExtension.class, closure);
```

Comment:
`yaml` is using `JsonExtension.NAME` instead of `YamlExtension.NAME`

Reply:
Pfff. Poor myself xD. Thanks for the 2 additional eyes.